### PR TITLE
Jekyll blog template

### PR DESCRIPTION
**Pull request by ocefpaf**
@conda-forge/core we can build this locally with:

```shell
conda create --name JEKYLL rb-bundler compilers
conda activate JEKYLL
bundle exec jekyll serve
```

### CODE REVIEW

File: _posts/2019-12-06-cfep09.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+---
+layout: post
+title: conda-forge CFEP-09
```
Comment (ocefpaf) [inline _posts/2019-12-06-cfep09.md]: @CJ-Wright this is your blog post. Do you want to change the title?

Comment (CJ-Wright) [reply _posts/2019-12-06-cfep09.md]: Most likely, although not certain to what yet.

Comment (CJ-Wright) [inline _posts/2019-12-06-cfep09.md]:
```suggestion
title: Automatically Deployed ABI Migrations
```

Comment (ocefpaf) [reply _posts/2019-12-06-cfep09.md]: Fancy!

**Comment by parulc7** [comment]
@ocefpaf I have another idea
we can integrate the blog to the website of conda-forge as well, and make a dynamic blog instead of just a Jekyll based static blog. We can add an editor based dashboard as well for admins to add posts or smth, if that seems feasible to you.

**Comment by parulc7** [comment]
However, that would take a lot of time. The quickest way to get going is using a static website generator for now, while you guys assign someone to build a dynamic backend based blog - 

You'll need -
1. Authentication for Admin Panel
2. Editor Integration with the Dashboard 

It will be a simple project and I would be happy to take that up as well if you guys allow me to.

**Comment by ocefpaf** [comment]
Let's try a simpler jekyll approach first and see how that works for us. I'm having difficulty getting people to review this PR so I'm self-merging it to see if we need to make adjustments to get it published. If it works we can expand with new posts and improvements.

**Comment by jakirkham** [comment]
Thanks for doing this @ocefpaf! So what are the next steps for us?

**Comment by ocefpaf** [comment]
I'll be away for a few days but the first step is to check the Jekyll configuration b/c the page is a bit wonky. I'm probably missing something. If anyone has experience with Jekyll I appreciate if they could take a look at it.

**Pull request merged**